guides/common/modules/proc_preparing-for-using-external-databases.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="preparing-for-using-external-databases_{context}"]
+= Preparing {Project} for using external databases
+
+By default, the {Project} installation process includes installing a PostgreSQL database on the same host as {ProjectServer}.
+However, in certain {Project} deployments, using external databases instead of the default local databases can help with the server load or have other benefits.
+
+.Prerequisites
+* You have considered whether using an external database is beneficial for your use case. 
+For more information, see {AdministeringDocURL}postgresql-as-an-external-database-considerations_admin[PostgreSQL as an external database considerations] in _{AdministeringDocTitle}_.
+
+.Procedure
+. Set up a system to host the external database. 
+For more information, see {AdministeringDocURL}preparing-a-host-for-external-databases_admin[Preparing a host for external databases] in _{AdministeringDocTitle}_.
```

Comment:
This anchor doesn't work, at least for the Katello guide.

Reply:
Yes, turns out the entire section has been removed between 6.16 and 6.17: https://github.com/theforeman/foreman-documentation/pull/3167/files . To adjust, I've removed this step altogether: https://github.com/theforeman/foreman-documentation/pull/3985/commits/b36244d762eca41a2854c6e24e33ac7f14d88626

The procedure is now just a single step, but I daresay still works in terms of the UX.